### PR TITLE
feat: move events to top level of audit configuration

### PR DESCRIPTION
**Pull request by yquansah**
Move `events` to top level of the audit configuration.

### CODE REVIEW

File: internal/config/audit.go [hunk 1]
```diff
@@ -15,17 +15,17 @@ var _ defaulter = (*AuditConfig)(nil)
 type AuditConfig struct {
 	Sinks  SinksConfig  `json:"sinks,omitempty" mapstructure:"sinks"`
 	Buffer BufferConfig `json:"buffer,omitempty" mapstructure:"buffer"`
+	Events []string     `json:"events,omitempty" mapstructure:"events"`
 }
 
 // Enabled returns true if any nested sink is enabled
 func (c *AuditConfig) Enabled() bool {
-	return c.Sinks.LogFile.Enabled
+	return c.Sinks.LogFile.Enabled || c.Sinks.Webhook.Enabled
```
Comment (markphelps) [inline internal/config/audit.go:23]: wonder if we should add a test for this

**Review by markphelps — APPROVED** [review]
lgtm, one testing comment

File: internal/config/audit.go (hunk 1 above)
Comment (yquansah) [reply internal/config/audit.go:23]: @markphelps Like a test within the `config/` directory?

Comment (markphelps) [reply internal/config/audit.go:23]: yeah, dont worry about it though. im actually gonna cover it in my telemetry PR

**Pull request merged**